harness/determined/cli/job.py

```diff
@@ -100,6 +87,83 @@ def update(args: Namespace) -> None:
     )
 
 
+@authentication.required
+def process_updates(args: Namespace) -> None:
+    session = setup_session(args)
+    for arg in args.operation:
+        inputs = validate_operation_args(arg)
+        _single_update(session=session, **inputs)
+
+
+def _single_update(
+    job_id: str,
+    session: Session,
+    priority: str = None,
+    weight: str = None,
+    resource_pool: str = None,
+    behind_of: str = None,
+    ahead_of: str = None,
+) -> None:
+    update = bindings.v1QueueControl(
+        jobId=job_id,
+        priority=priority,
+        weight=int(weight) if weight else None,
+        resourcePool=resource_pool,
+        behindOf=behind_of,
+        aheadOf=ahead_of,
+    )
+    bindings.post_UpdateJobQueue(session, body=bindings.v1UpdateJobQueueRequest([update]))
+    return
+
+
+def is_priority_rm(config: dict) -> bool:
+    try:
+        if config["resource_manager"]["scheduler"]["type"] == "fair_share":
```

Comment:
we have other schedulers besides `fair_share` e.g. round robin. so maybe it's better to check for priority specifically?